### ISSUE TITLE
Prototype should not contain error messages

### ISSUE THREAD

**Issue by lordjancso** [kristijanhusak/laravel4-form-builder#10]
See title

**Comment by kristijanhusak** [comment]
I agree, but as we discussed in https://github.com/kristijanhusak/laravel4-form-builder/issues/9 , errors should not be visible if the messages names are correct. I'll look into it anyway. Thanks for reporting.
